commands/admin/lock.py

```
@@ -5,6 +5,8 @@
 import utility
 from api import clear_channel_overwrites, save_channel_overwrites
 from localizer import tanjunLocalizer
+from utility import CommandInfo
```

Flake8 (7.3.0)

[error] 'utility.CommandInfo' imported but unused

(F401)